Lib/subprocess.py

```diff
@@ -908,6 +913,8 @@ def __exit__(self, exc_type, value, traceback):
             self.wait()
 
     def __del__(self, _maxsize=sys.maxsize, _warn=warnings.warn):
+        if _mswindows:
+            return
```

Comment:
Maybe explain why nothing needs to be done in Windows by repeating or referencing the comment in `__init__`.